docs/observability/metrics.md

```diff
@@ -114,6 +114,9 @@ Virtual Machine last transition timestamp to running status. Type: Counter.
 ### kubevirt_vm_starting_status_last_transition_timestamp_seconds
 Virtual Machine last transition timestamp to starting status. Type: Counter.
 
+### kubevirt_vm_vnic_info
+Information about Virtual Machines vNIC interfaces. Type: Gauge.
```

Comment:
@avlitman 
Details of Virtual Machine (VM) vNIC interfaces, such as vNIC name, binding type, network name, and binding plugin in use for each vNIC defined in the VM's configuration.

Reply:
fixed